src/scripts/main.js

```diff
@@ -1,3 +1,224 @@
 'use strict';
 
-// write code here
+const DESC = 'desc';
+const ASC = 'asc';
+const HEADERS = ['name', 'position', 'office', 'age', 'salary'];
+const OFFICES = [
+  'Tokyo',
+  'Singapore',
+  'London',
+  'New York',
+  'Edinburgh',
+  'San Francisco',
+];
+
+const itemCollection = document.querySelectorAll('tbody tr');
+const tableHeader = document.querySelector('thead tr');
+const tableBody = document.querySelector('tbody');
+
+const items = [...itemCollection].map((item) => {
+  const collection = item.querySelectorAll('td');
+
+  return HEADERS.reduce(
+    (prev, current, index) => ({
+      ...prev,
+      [current]: collection[index].innerText,
+    }),
+    {},
+  );
+});
```

Comment:
This operation of mapping table rows to objects can be moved to a separate function to improve code readability and maintainability. The function can be named something like 'mapTableRowsToObjects'.